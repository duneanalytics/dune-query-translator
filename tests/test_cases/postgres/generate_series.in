SELECT generate_series('2023-01-01'::timestamp, '2023-02-01'::timestamp, '1 day')