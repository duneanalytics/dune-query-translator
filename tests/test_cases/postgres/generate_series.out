SELECT
  col
FROM UNNEST(SEQUENCE(
  TRY_CAST('2023-01-01' AS TIMESTAMP),
  CAST(CAST(CAST(TRY_CAST('2023-02-01' AS TIMESTAMP) AS TIMESTAMP) AS TIMESTAMP) AS TIMESTAMP),
  INTERVAL '1' day
)) AS _u(col)